content/news/_index.md

```diff
@@ -1,6 +1,8 @@
 ---
 title: "News"
-seo_title: "News | Jakarta EE | The Future of Cloud Native Java"
```

Comment:
I think we want to keep seo_title and description based on the SEO report that was done recently. 

If the content is no longer relevant, I would recommend that we update it instead of replacing it.

Reply:
will add seo title 👍

Reply:
done